components/EditableTagList.vue

```diff
@@ -74,32 +90,53 @@
         @blur="blurInput"
       />
       <div
-        v-if="shouldShowInput"
         :class="[
+          'flex',
+          'justify-center',
+          'items-center',
+          'box-border',
           'top-0',
           'left-0',
-          'min-h-[30px]',
+          'h-[30px]',
+          { 'border-[1px] border-red': errorMessage },
           'bg-shade-gray',
           'py-[6px]',
           'px-[20px]',
-          'mb-[4px]',
           'text-[12px]',
           'rounded-[16px]',
         ]"
       >
         {{ currentInput }}
       </div>
+      <span
+        v-if="errorMessage"
+        :class="[
+          'block',
+          'self-end',
+          'text-[red]',
+          'text-[10px]'
+        ]"
+      >
+        {{ errorMessage }}
+      </span>
     </form>
   </div>
 </template>
 
 <script lang="ts">
-import { Vue, Component, Model, Watch } from 'vue-property-decorator'
+import { Vue, Component, Model, Watch, Prop } from 'vue-property-decorator'
 
 @Component
 export default class EditableTagList extends Vue {
   @Model('change', { type: Array, default: () => [] }) value!: Array<string>
 
+  // Character limit for a tag
+  @Prop(Number) readonly wordLimit!: number | undefined
```

Comment:
`charactersLimit` is more precise of the meaning, should provide default as well?